lib/const.rb

```diff
@@ -76,9 +76,7 @@
   case ARCH
   when 'i686'
     CREW_KERNEL_VERSION = '3.8'
-  when 'aarch64', 'armv7l'
-    CREW_KERNEL_VERSION = '5.10'
-  when 'x86_64'
+  when 'aarch64', 'armv7l', 'x86_64'
```

Comment:
How about just `else`?

Reply:
Doing that triggers some additional rubocop demands, but I let it autocorrect those